.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  env: {
+    worker: true,
+    es6: true,
+  },
+  rules: {
+    'no-unused-vars': ['error', { argsIgnorePattern: 'request' }],
```

Comment:
I don't think we should ignore this simply because we don't use it - we should probably just `console.log(request)` in the template so it's used instead